src/components/gallery/arts/arts.tsx

```diff
@@ -168,19 +168,19 @@ const artworkImages = [
 
 export default function Arts() {
   return (
-    <div className={styles.mainContent}>
-      <div className={styles.galleryBoxInner}>
-        {artworkImages.map((art) => (
-          <div className={styles.artworkBox} key={art.id}>
-            <div className={styles.artwork}>
-              <Link to={`/gallery/${art.id}`}>
-                <img src={art.src} alt={art.artistName} />
-              </Link>
-            </div>
-            <span>{art.artistName}</span>
+    <div className='flex flex-col justify-center items-center w-[1080px]'>
+    <div className='flex flex-wrap justify-center gap-[58px]'>
+      {artworkImages.map((art) => (
+        <div className='w-[320px] h-[320px] bg-[var(-btn-dark)]' key={art.id}>
```

Comment:
```suggestion
        <div className='w-[320px] h-[320px] bg-black' key={art.id}>
```
저희 기본 black이 `#333` 이므로 이렇게 사용하시면 됩니다

Reply:
++ 여기도 `id`가 아니라 `artNo`가 될 것 같아요. 참고해두시면 좋을 것 같습니다.

Reply:
넵